packages/autorest.typescript/test/rlcIntegration/generated/multipleInheritanceRest/samples-dev/putCatSample.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createMultipleInheritanceRestClient, {
+  PutCatParameters
+} from "@msinternal/multiple-inheritance-rest";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to call operation PutCat
+ *
+ * @summary call operation PutCat
+ */
+async function putCatSample() {
+  const client = createMultipleInheritanceRestClient();
+  const options: PutCatParameters = {
+    body: { likesMilk: true, name: "{Your name}", meows: true, hisses: true },
```

Comment:
Can we format the body part here?  Just curious that in some cases, it is  well formatted.

Reply:
We have formatted these files in a unified manner, here the total len is smaller than 80 so it would be in one line.

```typescript
export const prettierTypeScriptOptions: Options = {
  parser: "typescript",
  arrowParens: "always",
  bracketSpacing: true,
  endOfLine: "lf",
  printWidth: 80,
  semi: true,
  singleQuote: false,
  tabWidth: 2
};
```